src/Discord/Internal/Types/Embed.hs

```diff
@@ -122,13 +129,12 @@ instance FromJSON Embed where
           <*> o .:? "footer"
           <*> o .:? "color"
           <*> o .:? "timestamp"
-          <*> o .:? "type"
           <*> o .:? "video"
           <*> o .:? "provider"
 
 
 data EmbedThumbnail = EmbedThumbnail
-  { embedThumbnailUrl :: Maybe T.Text
+  { embedThumbnailUrl :: T.Text
   , embedThumbnailProxyUrl :: Maybe T.Text
   , embedThumbnailHeight :: Maybe Integer
   , embedThumbnailWidth :: Maybe Integer
```

Comment:
Hm. The docs  do say the thumbnail url is always present. Weird. 

<https://discord.com/developers/docs/resources/channel#embed-object-embed-image-structure>